proposals/collection-literals.md

```diff
@@ -555,6 +513,107 @@ Not having a *known length* does not prevent any result from being created. Howe
 
         The counts passed to `CreateArray` are used to provide a starting size hint to prevent wasteful resizes.
 
+## Dictionaries
+
+[*Constructible collection types*](#constructible-collection-types) is updated to include dictionaries.
+
+The following are _constructible collection types_ that can be used to construct collection literals.
+
+* ...
+* `Dictionary<TKey, TValue>`
+* Interface types `I<TKey, TValue>` implemented by `Dictionary<TKey, TValue>` (e.g. `IDictionary<TKey, TValue>`, `IReadOnlyDictionary<TKey, TValue>`)
+
+_Note that `Dictionary<TKey, TValue>` does not include a public `Add()` instance method with a single parameter, so the type is not constructible as a *collection initializer*._
+
+_Use `this[int index] { set; }` rather than `Add()` when constructing a dictionary for consistent set rather than add semantics._
```

Comment:
I'm not sure what this means. Why would we want collection literals to have different observable behavior than collection initializers today?

Reply:
primarily for the scenario of:

```c#
var d3 = [..d1, ..d2];
```

Which teh working group found compelling.  The feeling was that being able to merge dictionaries, with "last one wins" semantics is both reasonable and desirable, so we'd prefer 'lenient overwrite' semantics, versus 'restrictive throw' semantics.